app/build.gradle

```diff
@@ -89,6 +89,9 @@ dependencies {
     compile 'io.branch.sdk.android:library:1.+'
 
     compile('com.twitter.sdk.android:twitter:1.12.1@aar') {
+        transitive = true
+    }
+    compile('com.crashlytics.sdk.android:crashlytics:2.6.4@aar') {
```

Comment:
I've deleted the crashlytics dependency because i don't want to mess my crash data on the production dashboard. So...sorry but i will close this PR